pkg/aws/v1alpha3/awscontrolplane/validate_awscontrolplane.go

```diff
@@ -88,10 +88,6 @@ func (v *Validator) ValidateUpdate(request *admissionv1.AdmissionRequest) (bool,
 	if err != nil {
 		return false, microerror.Mask(err)
 	}
-	err = v.AZUnique(awsControlPlane)
```

Comment:
The thing is it will contain 3 AZs now when we add the third one, and we have to allow repeating them because we don't want to migrate the CP to also use the 3rd AZ

Reply:
Ok make sense, then let's drop it but don't forget the CHANGELOG and the README file that we don't check it anymore 🙂